src/test/java/automation/utils/WaitForElement.java

```diff
@@ -0,0 +1,20 @@
+package automation.utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WaitForElement {
+
+    public static void wait(WebDriver driver, By pathToElement) {
+        WebDriverWait wait = new WebDriverWait(driver, 3);
```

Comment:
What is '3'? Do not hard code values. Use constants instead. BTW, you were forced to repeat yourself here, because not using constants.

Reply:
3 is time out - I sent this to environment.properties. Also remove 'driver' field from method.